eZ/Bundle/EzPublishCoreBundle/Command/NormalizeImagesPathsCommand.php

```diff
@@ -146,25 +179,25 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         return 0;
     }
 
-    private function updateImagePath(int $fieldId, string $oldPath, string $newPath, SymfonyStyle $io): void
-    {
+    /**
+     * @param resource $inputStream
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
```

Comment:
There's one more exception to report here:
```suggestion
     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
```